Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java

```diff
@@ -0,0 +1,120 @@
+package com.dwg_karrier.roys;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/* Add on to listActivity
+* Use getTitle(), getContent(), getWordCount() */
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7-javadoc
Unlike the usual comments, there's a comment with certain format called Javadoc.
It's usually used on description of a class or method, and has the bit different shape as follows.
Upper link is about the Javadoc style in Google, and here's the Wikipedia link for the real Javadoc: https://en.wikipedia.org/wiki/Javadoc